service/RoleBandService.ts

```diff
@@ -0,0 +1,14 @@
+import { RoleBandResponse } from "../model/RoleBandResponse";
+const axios = require('axios');
+
+axios.defaults.baseURL = process.env.API_URL;
```

Comment:
When echoing API_URL it comes out correctly but if i console.log it returns as 'undefined', I've sourced .zshrc so I'm unsure how to fix this

Reply:
Rather than using axios.defaults here let's create a variable that we can access below  

```suggestion
const baseURL = process.env.API_URL;
```